packages/components/select/src/useSelect.ts

```diff
@@ -603,10 +604,15 @@ export const useSelect = (props, states: States, ctx) => {
     }
   }
 
+  const getLastNotDisabledIndex = (value) =>
+    value.findLastIndex((it) => !states.disabledOptions.has(it))
```

Comment:
## Suggest

It seems better to use lodash's findLastIndex than bypassing it with a mock .

If you accept this, please delete all test code changes.

```ts
import {
  findLastIndex,
} from 'lodash-unified'
```

Reply:
Oh wait! I checked and the test still seems to be broken.

Reply:
hello~ test code have been add.